ajax/libs/html5-device-mockups/package.json

```diff
@@ -0,0 +1,36 @@
+{
+  "name": "html5-device-mockups",
+  "filename": "device-mockups.min.css",
+  "description": "HTML5 mockups of popular devices, to showcase your portfolio and spice up your website.",
+  "keywords": [
+    "html5",
+    "device",
+    "mockups",
+    "demo",
+    "portfolio"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/pixelsign/html5-device-mockups.git"
+  },
+  "homepage": "https://pixelsign.github.io/html5-device-mockups/",
+  "license": "MIT",
```

Comment:
I got the reply from the author. He said they are using MIT license.

Reply:
Great.
would you please also add `cc cdnjs/cdnjs#12298` in your issue of the upstream?
This will help reviewers know the current license which you got the request from the opened issue of upstream.

Thanks.